test/e2e/storage/utils/pod.go

```diff
@@ -49,7 +49,7 @@ func StartPodLogs(f *framework.Framework, driverNamespace *v1.Namespace) func()
 
 	ns := driverNamespace.Name
 
-	podEventLog := ginkgo.GinkgoWriter
+	var podEventLog io.Writer = ginkgo.GinkgoWriter
```

Comment:
The code is okay, but the commit message says "Migrate to GinkgoWriter" which doesn't match the code. Perhaps just say "Adapt to new type of GinkgoWriter"?